_scss/main.scss

```diff
@@ -165,8 +165,9 @@ nav {
         margin-left: 30%;
         background-color: map-get($red, accent-1);
 
-        .input-field {
+        .input-field label {
             height: 100%;
+            top: 1.3rem;
```

Comment:
Static setting isn't necessary. You should avoid doing such assignments and try to look at suitable classes from materialize-css which would get you the desired behaviour. I found `label-icon` class in the css-console which makes it align properly. There is a "transform" which needs to be fixed, by setting `transform: none`.

Reply:
Ok I will keep these points in mind.
Thanks for help